custom_metrics/almanac.js

```diff
@@ -12,32 +12,263 @@
 // 3. Test your change by following the instructions at https://github.com/HTTPArchive/almanac.httparchive.org/issues/33#issuecomment-502288773.
```

Comment:
Item 2 above is obsolete now that we're stringifying the whole response. Can you replace that guidance with 2a?

Reply:
I can't see to what code line your remark refers.

Reply:
I'm unable to comment on a line that you haven't modified. The instruction on the lines above to jsonify objects/arrays is outdated, so it should be removed. If you don't do it in this PR I can submit a follow-up.

Reply:
Ok, done.